src/main/java/com/newrelic/experts/jenkins/extensions/JenkinsMasterEventPeriodicWork.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2019 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 
+ */
+
+package com.newrelic.experts.jenkins.extensions;
+
+import com.google.inject.Inject;
+
+import com.newrelic.experts.client.model.JenkinsMasterEvent;
+import com.newrelic.experts.jenkins.EventRecorder;
+import hudson.Extension;
+import hudson.model.PeriodicWork;
+import jenkins.model.Jenkins;
+
+import java.util.Arrays;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * A Jenkins {@link PeriodicWork} object that adds JenkinsMaster events for
+ * event collection, by default running once every minute.
+ * 
+ * @author Dan Alvizu (alvizu@gmail.com)
+ */
+@Extension
+public class JenkinsMasterEventPeriodicWork
```

Comment:
@dalvizu to stay consistent with the naming of `EventCollectorWork`, can we take the `Periodic` out of this class name please?